python/sdk/prelude_sdk/models/codes.py

```diff
@@ -220,16 +220,17 @@ def mapping(cls):
             ],
         }
 
-class PartnerSyncJobType(Enum, metaclass=MissingItem):
+class BackgroundJobTypes(Enum, metaclass=MissingItem):
     INVALID = -1
-    POLICY_EVALUATION = 1
+    UPDATE_SCM = 1
     DEPLOY_PROBE = 2
     OBSERVED_DETECTED = 3
     PRELUDE_ENDPOINT_SYNC = 4
+    EXPORT_SCM = 5
```

Comment:
any reason this needs to be SCM-specific vs just "EXPORT" ?

Reply:
i can change it to be export, if we ever need to have different export methods in the future we can always add more

Reply:
i guess the only thing this is good for is in the ListBackgroundJobs response really. and would you want literally all your exports showing up in a clump or would we want to group by export types

Maybe EXPORT_SCM was better. sorry. sometimes i ask questions before i really think about them